### PR TITLE
Create Appointment Completed

### PR DESCRIPTION
**Pull request by VireshReddy123**

### CODE REVIEW

File: src/main/java/org/automation/pageObjects/ScheduleAppointmentPage.java [hunk 1]
```diff
@@ -120,14 +187,40 @@ public void appointmentDateSelecting(int Type) throws InterruptedException {
...
             clickOnAssesmentType(Type);
         }
-        public void enteringClientDetails(String firstName,String lastName,int gradeType,String schoolTypeOption,String cellNumberText,String phoneNumberText){
-            enterFirstName(firstName);
-            enterLastName(lastName);
+        public String enteringClientDetails(String CustomerFirstName, String CustomerLastName, int gradeType, int schoolTypeOption, String cellNumber , String EmailAddress, String reasonForCallText, String address1Text, String address2Text, String cityText, String stateText, String zipCodeText, String testAmountText, String enterAmountText) throws InterruptedException {
+            enterFirstName(CustomerFirstName);
+            enterLastName(CustomerLastName);
+            Thread.sleep(2000);
+            enterInDateField( "11-11-1111");
```
Comment (SKaur2400) [inline src/main/java/org/automation/pageObjects/ScheduleAppointmentPage.java:194]: remove hard coding

File: src/main/java/org/automation/utilities/RandomStrings.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
+package org.automation.utilities;
+
+public class RandomStrings {
+    public static String RequiredString(int n)
```
Comment (SKaur2400) [inline src/main/java/org/automation/utilities/RandomStrings.java:4]: use camel case for method name

File: src/main/java/test/ScheduleAppointmentTest.java [hunk 3]
```diff
@@ -29,7 +35,17 @@ public void appointmentCalender() throws InterruptedException {
...
+        ScheduleAppointmentPage schedule = new ScheduleAppointmentPage();
+         CustomerFirstName = "Michel" + RandomStrings.RequiredCharacters(2);
+         CustomerLastName = "Marsh" + RandomStrings.RequiredCharacters(2);
+         EmailAddress =  CustomerFirstName+"@yopmail.com";
+        schedule.enteringClientDetails( CustomerFirstName,CustomerLastName,2,2,"4567892658",EmailAddress,"Math","NSW"," Tasmania"," Barkers Creek","South Australia","5422","1200","1000");
+    }
+    @Test(priority = 3,enabled = true)
+    public void filterCreatedAppointment() throws InterruptedException {
+        ScheduleAppointmentPage schedule = new ScheduleAppointmentPage();
+        schedule.selectCreatedAppointment(CustomerFirstName);
```
Comment (SKaur2400) [inline src/main/java/test/ScheduleAppointmentTest.java:48]: add atleast one assertion in each testcase

File: src/main/java/org/automation/pageObjects/ScheduleAppointmentPage.java (hunk 1 above)
Comment (VireshReddy123) [reply src/main/java/org/automation/pageObjects/ScheduleAppointmentPage.java:194]: s mam removed

File: src/main/java/org/automation/utilities/RandomStrings.java (hunk 2 above)
Comment (VireshReddy123) [reply src/main/java/org/automation/utilities/RandomStrings.java:4]: k mam

**Pull request merged**